### ISSUE TITLE
Clientside lag caused by vines

### ISSUE THREAD

**Issue by LoneDev6** [imDaniX/Dangerous-Caves-2#35]
**Describe the bug**
Basically the client lag too much because vines are resource intensive on the clientside.

**Screenshots or logs**
![image](https://user-images.githubusercontent.com/27242001/97082681-c7f36300-160b-11eb-83f9-795eb0324b15.png)

**Additional information**
Even with a good PC

**Solution**
Disable vines by default or remove them from the plugin

**Comment by imDaniX** [comment]
I don't think that's a good way to fix the problem. Maybe I should add some threshold for every type of aging instead of total percentage(or just use both). Also, maybe random vine removing as a part of aging...

**Comment by LoneDev6** [comment]
Well idk if there is a clean solution to solve this, but you see, after some gameplay hours you see that vines, mushrooms and stone buttons are basically covering the whole underground parts of the map which is awful and makes the client lag as hell

(I'm not insulting your plugin, the idea is great, the problem is that Minecraft is shitty on performance)

**Comment by imDaniX** [comment]
Yeah, ikr, I just don't want to disable it by default. Want to collect some ideas if people have any. As for now I'll stick with vines removing + per-type threshold.. but we'll see.

**Comment by LoneDev6** [comment]
maybe place bushes and grass instead of vines
(place them without applying physics or they will pop out)

**Comment by Kenaestic** [comment]
> 
> 
> Yeah, ikr, I just don't want to disable it by default. Want to collect some ideas if people have any. As for now I'll stick with vines removing + per-type threshold.. but we'll see.

I kinda assumed this fell under
`# Maximal percentage of aged blocks per chunk
    percentage: 10`

**Comment by imDaniX** [comment]
Ehh, misspelled issue id.
